Makefile

```diff
@@ -0,0 +1,27 @@
+start: ## Start the docker containers
+	@echo "Starting the docker containers"
+	@docker-compose up -d
+	@echo "Containers started - http://localhost:8000"
+
+stop: ## Stop Containers
+	@docker-compose down
+
+build: ## Build Containers
+	@docker-compose build
+
+ssh: ## SSH into running web container
+	docker-compose exec web ash
```

Comment:
what is `ash`?

Reply:
[Almquist Shell](https://en.wikipedia.org/wiki/Almquist_shell) which ships with Alpine Docker images. Pulled from here: https://stackoverflow.com/a/35689633/8207

Reply:
cool! thanks :)